api/models/Developer.js

```diff
@@ -1,9 +1,13 @@
 module.exports = {
   attributes: {
-    github_id: 'integer',
-    username : 'string',
-    profile  : {
+    github_id   : 'integer',
+    username    : 'string',
+    profile     : {
       model: 'profile'
+    },
+    organization: {
```

Comment:
Users can belong to multiple organizations. Also, via is not needed for a oneTo* join.